src/proto/thread_telemetry.proto

```diff
@@ -496,6 +496,19 @@ message TelemetryData {
     optional uint32 global_unicast_address_count = 7;
   }
 
+  // Message to indicate the information of external routes in network data.
+  message ExternalRoutes {
+    // Indicates whether the a zero-length prefix (::/0) added from this BR
+    optional bool has_default_route_added = 1;
+
+    // Indicates whether the a ULA prefix (fc00::/7) added from this BR
+    optional bool has_ula_route_added = 2;
+
+    // Indicates whether the other prefixes (other than "::/0" or "fc00::/7") added
+    // from this BR. (BR is a managed infrastructure router).
+    optional bool has_others_route_added = 3;
```

Comment:
```suggestion
    optional bool has_other_route_added = 3;
```

Reply:
ack

Reply:
Thanks for raising this! My thinkings are:
1. "others" acts as a pronoun here, referring to all routes other than "default" and "ULA"
2. "has_XXX_route_added" maintains grammatical consistency with the other fields